### PR TITLE
Console - Finish some page migration

### PR DESCRIPTION
**Pull request by ThibaudAV**
## Issue

https://gravitee.atlassian.net/browse/APIM-3989
https://gravitee.atlassian.net/browse/APIM-4011

## Description

Finish some page migration

## Additional context

🏗️ Your changes can be tested here and will be available soon:
      Console: [https://pr.team-apim.gravitee.dev/7135/console](https://pr.team-apim.gravitee.dev/7135/console)
      Portal: [https://pr.team-apim.gravitee.dev/7135/portal](https://pr.team-apim.gravitee.dev/7135/portal)
      Management-api: [https://pr.team-apim.gravitee.dev/7135/api/management](https://pr.team-apim.gravitee.dev/7135/api/management)
      Gateway v4: [https://pr.team-apim.gravitee.dev/7135](https://pr.team-apim.gravitee.dev/7135)
      Gateway v3: [https://pr.gateway-v3.team-apim.gravitee.dev/7135](https://pr.gateway-v3.team-apim.gravitee.dev/7135)

---

📚&nbsp;&nbsp;View the storybook of this branch [here](https://612657caa8e859003a8a6430-rfcdwfvgwe.chromatic.com)

### CODE REVIEW

**Comment by phiz71** [comment]
LGTM but it will deserve the "ready-to-test" label and/or screenshots to be sure everything is ok

**Comment by phiz71** [comment]
I've played with the PR environment, and it looks ok.
However, I've noticed that the TopAPIs screens calls the mAPI v1 to search for APIs. As a consequence, we can only put v2 APIs in the top APIs list.
Should call the mAPI v2 ?

**Comment by ThibaudAV** [comment]
Yes, I hadn't noticed. For your information @pes-spyro-soft-com i'll take the fix into account and rework this part. So that it can do an autocomplete search. without trying to load all the apis

File: gravitee-apim-console-webui/src/management/settings/top-apis/add-top-apis-dialog/add-top-apis-dialog.component.ts [hunk 1]
```diff
@@ -39,30 +38,30 @@ export type AddTopApisDialogResult = Api;
 })
 export class AddTopApisDialogComponent implements OnInit {
   public searchApiControl: FormControl<string | Api> = new FormControl('');
-  public apis: Api[] = [];
+  // public apis: Api[] = [];
```
Comment (phiz71) [inline gravitee-apim-console-webui/src/management/settings/top-apis/add-top-apis-dialog/add-top-apis-dialog.component.ts]: to remove

File: gravitee-apim-console-webui/src/management/settings/top-apis/add-top-apis-dialog/add-top-apis-dialog.component.ts [hunk 2]
```diff
@@ -39,30 +38,30 @@ export type AddTopApisDialogResult = Api;
...
+      debounceTime(300),
+      distinctUntilChanged(),
+      switchMap((term: string) =>
+        this.apiService.search({
+          query: term,
+        }),
+      ),
       tap(() => (this.isApiSelected = false)),
-      map((value: string): Api[] => (value ? this._filter(value) : [])),
+      map((apisRspoinse) => apisRspoinse.data),
```
Comment (phiz71) [inline gravitee-apim-console-webui/src/management/settings/top-apis/add-top-apis-dialog/add-top-apis-dialog.component.ts]: apisRspoinse => apisResponse

**Review by phiz71 — CHANGES_REQUESTED** [review]
LGTM, just a few changes to do before approval

**Review by phiz71 — APPROVED** [review]

**Pull request merged**